tcpdf.php

```diff
@@ -1831,11 +1837,11 @@ class TCPDF {
 	 * @param $unicode (boolean) TRUE means that the input text is unicode (default = true)
 	 * @param $encoding (string) Charset encoding (used only when converting back html entities); default is UTF-8.
 	 * @param $diskcache (boolean) DEPRECATED FEATURE
-	 * @param $pdfa (boolean) If TRUE set the document to PDF/A mode.
+	 * @param $pdfa (string) If pdfa set the document to PDF/A mode, if pdfx set the document to PDF/X mode or none.
```

Comment:
The parameter name should now be $pdfmode
It would be helpful if specifically documented that the mode is PDF/X-3 as others could assume it is PDF/X-1 for example